config/locales/idv/en.yml

```diff
@@ -185,7 +185,7 @@ en:
         review: Re-enter your %{app_name} password to protect your data
     troubleshooting:
       headings:
-        are_you_near: 'Are you near Washington, D.C.?'
+        are_you_near: 'Are you located near Washington, D.C. <br/>or Baltimore, MD?'
```

Comment:
Can we check if the line break element here may have some unintended impact on accessibility, such as a pause when reading? From what I recall, it may. We could consider adding an `aria-hidden` attribute to have it ignored by assistive technology. (This would need to be added as part of the `formatHTML` mapping, like in the example in the new documentation introduced with #7034).